Application/Services/GroupAuthorizationService.cs

```diff
@@ -0,0 +1,25 @@
+using Application.Exceptions;
+using Application.Interfaces;
+using Domain.Interfaces;
+
+namespace Application.Services;
+
+/// <summary>
+/// Provides services to ensure that a user is a member of a specific group.
+/// </summary>
+/// <param name="groupRepository">The repository for accessing group data.</param>
+public class GroupAuthorizationService(IGroupRepository groupRepository) : IGroupAuthorizationService
```

Comment:
Średnio dokładny ten komentarz, serwis nie dostarcza serwisów.
```suggestion
/// <summary>
/// Provides authorization checks related to group membership.
/// </summary>
/// <param name="groupRepository">The repository for accessing group data.</param>
public class GroupAuthorizationService(IGroupRepository groupRepository) : IGroupAuthorizationService
```